src/main/java/com/onevizion/scmdb/dao/DdlDao.java

```diff
@@ -23,7 +23,9 @@ public class DdlDao extends AbstractDaoOra {
             "   or object_type = 'VIEW'\n" +
             "   or object_type = 'PACKAGE'\n" +
             "   or object_type = 'PACKAGE BODY'\n" +
-            "   or object_type = 'TRIGGER'\n";
+            "   or object_type = 'TRIGGER'\n" +
+            "   or object_type = 'TYPE'\n" +
```

Comment:
Need to ignore types started with `T$`, it is internal autogenerated types. Please update the code to exclude it

Reply:
done

Reply:
what about type body?

Reply:
Sorry, added